lib/chitter.rb

```diff
@@ -0,0 +1,41 @@
+require 'date'
+require 'pg'
+require './lib/database_connection.rb'
+
+class Chitter
```

Comment:
I think Chitter could have a different name, I'm not sure what object Chitter is representing from the user story. Maybe a Tweet or a Peep object would be more readable?